src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs

```diff
@@ -64,6 +67,7 @@ public void Setup_FromLineBiggerThanToLine_HasNoReturn()
         [Test]
         public void Setup_FromLineIsSmallerThanOne_HasNoReturn()
         {
+            // Arrange? Act?
```

Comment:
Split into these certain steps not just write oneliners.

Reply:
The naming is misleading and it's eerly similar to Rajmunds method naming. Actually in the name you say `HasNoReturn` but in the method you expect an `ArgumentException`. Rename your method accordingly

Reply:
What about that case when the `toLine` parameter is less than 0 or more than the length of the file? What are the expected behavior in those cases?